### ISSUE TITLE
Bug in JavaScript texture decoding

### ISSUE THREAD

**Issue by mlenta** [google/draco#80]
Hi,

I found that, for some mesh, the texture mapping, decoded from Draco JavaScript API, will be weird. However, the C++ decoding result is correct. And I have updated my git repository.

The failed mesh: https://raw.githubusercontent.com/mlenta/reality/master/assets/avatar.obj
The texture: https://raw.githubusercontent.com/mrdoob/three.js/master/examples/models/collada/VWS_B_Male2-2.jpg
The material file: https://raw.githubusercontent.com/mlenta/reality/master/assets/avatar.obj.mtl 

And my result of JS decoding is like this.
![xx.jpg](https://raw.githubusercontent.com/mlenta/reality/master/assets/xx.jpg) 
The texture mapping of this avatar's face is wrong.

**Comment by ondys** [comment]
Thanks for the report. Can you also provide the encoded .drc file, or the settings that you used during encoding (if they are different from defaults)?

**Comment by mlenta** [comment]
I just used default settings:
+ Compression level is 7. 
+ Positions quantization is 14 bits. 
+ Texture coordinates quantization is 12 bits. 
+ And normals quantization is 10 bits. 

Here is the encoded .drc file if it may help:
https://github.com/mlenta/reality/raw/master/assets/avatar.drc

I am using Linux Mint 18 Cinnamon 64-bit. Inspired by your reply, I set texture coordinates quantization to 8 bits, it works well. Thank you :+1:  @ondys
